demos/point_discharge3d/setup.py

```diff
@@ -22,9 +22,8 @@
     p. 134 (2014).
```

Comment:
In Line 102: `/home/firedrake/firedrake/src/firedrake/firedrake/constant.py:73: FutureWarning: Giving Constants a domain is no longer supported. Instead please create a Function in the Real space.`

And just for info, Hessian and GO raise
```
*** buffer overflow detected ***: terminated

 Unexpected error:  *** potential lack of memory.
 ```

Reply:
This one works for me - I think it's just a matter of how much RAM you have as it's a pretty heavy test case.